unified_planning/model/problem.py

```diff
@@ -300,6 +305,38 @@ def get_static_fluents(self) -> Set["up.model.fluent.Fluent"]:
                     static_fluents.remove(e.fluent.fluent())
         return static_fluents
 
+    @property
+    def epsilon(self) -> Optional[Fraction]:
+        """
+        This parameter has meaning only in temporal problems: it defines the minimum
+        amount of time that can elapse between 2 temporal events. A temporal event can
+        be, for example, the start of an action, the end of an action, an intermediate
+        step of an action, a timed effect of the problem.
+
+        When None, it means that this minimum step is chosen by the Engine the Problem
+        is given to.
+        """
+        return self._epsilon
+
+    @epsilon.setter
+    def epsilon(self, new_value: Optional[Union[Real, Fraction]]):
+        if new_value is None:
+            pass
+        elif new_value < 0:
+            raise UPProblemDefinitionError(
+                "A negative epsilon is not valid and has no meaning!"
+            )
+        elif not isinstance(new_value, Fraction):
+            new_value = Fraction(float(new_value))
```

Comment:
Bad idea to go through float as an intermediate representation, as some fractionals would not be representable exactly:

```
>>>> print(Fraction(float(Fraction(1,10))))
3602879701896397/36028797018963968
```